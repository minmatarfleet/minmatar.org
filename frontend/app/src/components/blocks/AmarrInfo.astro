---
import { i18n } from '@helpers/i18n'
const { t } = i18n(Astro.url)

import StationInfo from '@components/blocks/StationInfo.astro';
---

<StationInfo
    system="Amarr"
    security="1.0"
    constellation="Throne Worlds"
    region="Domain"
    station_name="Amarr VIII (Oris) - Emperor Family Academy"
    station_description={t('amarr_hq_description')}
    slot="button"
/>