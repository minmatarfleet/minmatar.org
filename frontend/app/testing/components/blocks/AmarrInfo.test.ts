import { experimental_AstroContainer as AstroContainer } from "astro/container";
import { expect, test } from "vitest";
import AmarrInfo from "@components/blocks/AmarrInfo.astro";

test("AmarrInfo defaults", async () => {
  const container = await AstroContainer.create();
  const result = await container.renderToString(AmarrInfo, {});

  expect(result).toMatchSnapshot();
});
