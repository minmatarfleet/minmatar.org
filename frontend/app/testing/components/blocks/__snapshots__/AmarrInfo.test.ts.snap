// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AmarrInfo defaults 1`] = `"<div class="flex-block [ py-[var(--space-2xs)] animate-fade-in-up ] gap---space-2xs" slot="button" data-astro-cid-kj57dkoe="true" data-mod style="--flex_block_gap: var(--space-2xs);">  <div class="flex-inline [ !items-end !flex-nowrap ]" data-astro-cid-kj57dkoe="true" data-astro-cid-ud5fqf5k style="--flex_inline_gap: var(--space-3xs);--flex_inline_justification: flex-start;">  <h2 data-astro-cid-kj57dkoe>Amarr <span class="text-status-1" data-astro-cid-kj57dkoe>1.0</span></h2> <div class="flex-inline [ !items-end !flex-nowrap ]" data-astro-cid-kj57dkoe="true" data-astro-cid-ud5fqf5k style="--flex_inline_gap: var(--space-3xs);--flex_inline_justification: flex-start;">  <svg class="[ h-[17px] ]" width="6" height="9" viewBox="0 0 6 9" fill="none" xmlns="http://www.w3.org/2000/svg" data-astro-cid-kj57dkoe> <path d="M5.25 0L0 4.5L5.25 9V5.883L3.35175 4.5L5.25 3.117V0Z" fill="white" data-astro-cid-kj57dkoe></path> </svg> <h3 data-astro-cid-kj57dkoe><small data-astro-cid-kj57dkoe>Throne Worlds</small></h3> <svg class="[ h-[17px] ]" width="6" height="9" viewBox="0 0 6 9" fill="none" xmlns="http://www.w3.org/2000/svg" data-astro-cid-kj57dkoe> <path d="M5.25 0L0 4.5L5.25 9V5.883L3.35175 4.5L5.25 3.117V0Z" fill="white" data-astro-cid-kj57dkoe></path> </svg> <h3 data-astro-cid-kj57dkoe><small data-astro-cid-kj57dkoe>Domain</small></h3>  </div>    </div>   <h3 data-astro-cid-kj57dkoe>Amarr VIII (Oris) - Emperor Family Academy</h3> <small class="[ whitespace-nowrap ]" data-astro-cid-kj57dkoe>Minmatar Fleet Alliance HQ</small>  </div>  "`;
